src/css/App.css

```diff
@@ -0,0 +1,29 @@
+.testimonial-grid{
```

Comment:
This should rather be: `.App .testimonial-grid`
Just having the CSS declaration in a different file doesn't make it scoped to that file. Check the CSS output file that the browser receives and you'll see what I mean.